### DISCUSSION TITLE
Assembly and C low level and high level language called CASM.

### DISCUSSION THREAD

**Discussion by emullallphotography** [community/community#70320] [Programming Help]
### Body

Hi,

I am a beginner coder who wanted to build a programming language called CASM that offers both high-level and low-level capabilities 

Where can I learn enough computer engineering, C and NASM?
### Guidelines

- [X] I have read and understood this category's [guidelines](https://github.com/orgs/community/discussions/51384) before making this post.

**Comment by OSEight** [comment]
w3schools is good for C!
